api/PaymentRequestEvent.json

```diff
@@ -98,6 +98,54 @@
           }
         }
       },
+      "changePaymentMethod": {
+        "__compat": {
+          "spec_url": "https://w3c.github.io/payment-handler/#changepaymentmethod-method",
+          "support": {
+            "chrome": {
+              "version_added": "76"
+            },
+            "chrome_android": {
+              "version_added": "76"
+            },
+            "edge": {
+              "version_added": "79"
+            },
+            "firefox": {
+              "version_added": false
+            },
+            "firefox_android": {
+              "version_added": false
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": false
```

Comment:
Is this really right?

Reply:
Yeah, Opera doesn't support the payment APIs.

Reply:
Huh, OK :)